doc/classes/ProjectSettings.xml

```diff
@@ -244,6 +244,9 @@
 		<member name="application/config/windows_native_icon" type="String" setter="" getter="" default="&quot;&quot;">
 			Icon set in [code].ico[/code] format used on Windows to set the game's icon. This is done automatically on start by calling [method OS.set_native_icon].
 		</member>
+		<member name="application/run/delta_sync_after_draw" type="bool" setter="" getter="" default="false">
+			[b]Experimental[/b] Shifts the measurement of delta time for each frame to just after the drawing has taken place. This may lead to more consistent deltas and a reduction in frame stutters.
```

Comment:
```suggestion
			[b]Experimental.[/b] Shifts the measurement of delta time for each frame to just after the drawing has taken place. This may lead to more consistent deltas and a reduction in frame stutters.
```

Reply:
Just having a look at this... there are a number of other `Experimental` settings, should we change these for consistency?

Reply:
Have fixed this and the other occurrences in the ProjectSettings.